pinot-transport/src/main/java/com/linkedin/pinot/transport/scattergather/ScatterGatherImpl.java

```diff
@@ -479,12 +485,12 @@ private void runInternal() {
           if (errorMap != null && errorMap.containsKey(_server)) {
             errStr = errorMap.get(_server).getMessage();
           }
-          LOGGER.warn("Destroying invalid conn {}:{}", conn, errStr);
+          LOGGER.info("Destroying invalid conn {}:{}", conn, errStr);
```

Comment:
seems error. Are you taking approach of logging only non-recoverable issues as errors ?